DeepMRSeg/deepmrseg_apply.py

```diff
@@ -43,36 +39,30 @@ def read_flags(argv):
 
 	epilogTxt = '''Examples:
 	
-  ## Apply bmask model on single image (I/O OPTION 1)
-  deepmrseg_downloadmodel --model bmask
-  {prog} --task bmask --inImg sub1_T1.nii.gz --outImg sub1_bmaskseg.nii.gz
+  ## Apply dlicv model on single image (I/O OPTION 1)
+  deepmrseg_downloadmodel --model dlicv
+  {prog} --model dlicv --inImg sub1_T1.nii.gz --outImg sub1_DLICV.nii.gz
  
-  ## Apply wmlesion model (multi-modal with FL and T1 images)  on single subject (I/O OPTION 1)
-  deepmrseg_downloadmodel --model wmlesion
-  {prog} --task wmlesion --inImg sub1_FL.nii.gz --inImg sub1_T1.nii.gz --outImg sub1_wmlseg.nii.gz
-
-  ## Apply wmlesion model on multiple subjects using an image list (I/O OPTION 2)
-  deepmrseg_downloadmodel --model wmlesion
-  {prog} --task wmlesion --sList my_img_list.csv
+  ## Apply dlicv model on multiple subjects using an image list (I/O OPTION 2)
+  {prog} --model dlicv --sList my_img_list.csv
      with my_img_list.csv:
-       ID,FLAIR,T1,OutImg
-       sub1,/my/indir/sub1_FL.nii.gz,/my/indir/sub1_T1.nii.gz,/my/outdir/sub1_wmlseg.nii.gz
-       sub2,/my/indir/sub2_FL.nii.gz,/my/indir/sub2_T1.nii.gz,/my/outdir/sub2_wmlseg.nii.gz
+       ID,T1,OutImg
+       sub1,/my/indir/sub1_T1.nii.gz,/my/outdir/sub1_DLICV.nii.gz
+       sub2,/my/indir/sub2_T1.nii.gz,/my/outdir/sub2_DLICV.nii.gz
        ...
      
-  ## Apply roiseg model on all T1 images in the input folder (I/O OPTION 3)
-  deepmrseg_downloadmodel --model roiseg
-  {prog} --task roiseg --inDir /my/indir --outDir /my/outdir --inSuff _T1.nii.gz --outSuff _roiseg.nii.gz
+  ## Apply dlicv model on all T1 images in the input folder (I/O OPTION 3)
+  {prog} --model dlicv --inDir /my/indir --outDir /my/outdir --inSuff _T1.nii.gz --outSuff _DLICV.nii.gz
  
   '''.format(prog=exeName)
 
 	parser = _argparse.ArgumentParser( formatter_class=_argparse.RawDescriptionHelpFormatter, \
 		description=descTxt, epilog=epilogTxt )
 
-#	TASK
+#	TASK/MODEL
 #	===========
-	dirArgs = parser.add_argument_group( 'TASK')
-	dirArgs.add_argument( "--task", default=None, type=str, \
+	dirArgs = parser.add_argument_group( 'MODEL')
+	dirArgs.add_argument( "--model", default=None, type=str, \
```

Comment:
Why did we remove the '--task' parameter? That was much more intuitive.

Reply:
I also liked --task more, but changed it to --model to make it consistent with downloadmodel script, e.g.:
  deepmrseg_downloadmodel --model muse
  deepmrseg_apply --model muse

Reply:
I convert it back to task , following your suggestion